catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/GmtiMetacardType.java

```diff
@@ -58,5 +59,6 @@ public void initDescriptors() {
         descriptors.addAll(getDescriptors(CsdidaAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(CsexraAttribute.getAttributes()));
         descriptors.addAll(getDescriptors(HistoaAttribute.getAttributes()));
+        descriptors.addAll(getDescriptors(PiatgbAttribute.getAttributes()));
```

Comment:
Without looking deep at the NITF code, you may be refactor to do it in blueprint : https://github.com/codice/ddf/blob/master/catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L43